### ISSUE TITLE
Browser crash on launch

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#12413]
## Description 
Browser crash on Android 11

## Steps to reproduce 

   1. Install 1.18.18/21 nighlty 
   2. Open after install 
   3. Browser crash

## Actual result 
Browser crash

## Expected result
No crash

## Issue reproduces how often 
Easy

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? No
- Can you reproduce this issue with the current Play Store Beta version?  No crash on 1.17.54
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Phone OnePlus 6T 
   - Android version:11
  
## Brave version
1.18.18/1.18.21 nightly builds

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

**Comment by kjozwiak** [comment]
With my devices, I reproduced the crash only once on both Pixel 2 XL (Android 11) and Samsung S10+ (Android 10). Once I installed and launched , it crashed. But re-installing the same Nightly again always works for me after the first initial crash.

Seems like with @srirambv's case, he's always crashing on his Android 11 device.

**Comment by intrnl** [comment]
Android 10, always crashes one to three times before being able to browse (from a reboot or app close)

Constant crash when Tab Groups experiment is enabled (out of scope probably, just something worth noting)

**Comment by SergeyZhukovsky** [comment]
it was fixed in that one https://github.com/brave/brave-browser/issues/12431. Closing as duplicated.

**Comment by srirambv** [comment]
Marking as dupe based on https://github.com/brave/brave-browser/issues/12413#issuecomment-719942466 and moving out of milestone
